future-apps/payroll/contracts/Payroll.sol

```diff
@@ -43,53 +61,56 @@ contract Payroll is AragonApp { //, IForwarder { // makes coverage crash (remove
         uint64 lastPayroll;
         uint64 endDate;
         bool terminated;
```

Comment:
Do we really need `terminated`? Isn't it enough with `endDate`? See discussion [here](https://github.com/aragon/aragon-apps/pull/479#discussion_r232009250)

Reply:
Oh, not really, see more discussion [here](https://github.com/aragon/aragon-apps/pull/394#discussion_r201396637)